src/Refine/Flag_Check.cpp

```diff
@@ -362,26 +372,25 @@ bool Check_Curl( const int i, const int j, const int k,
 //                than the maximum angular resolution. Also, check if the cell is in Refined_Rad.
```

Comment:
- This description `Also, check if the cell is in Refined_Rad.` is obsolete
- Remove the trailing dot. Also in line 423 (the description in `Check_Radial()`)